grammars/jsdoc.cson

```diff
@@ -411,6 +435,120 @@
         ]
       }
     ]
+  # Default @param value, highlighted as JavaScript
+  'default-value':
+    'patterns': [
+      # Double-quoted string
+      {
+        'begin': '(")'
```

Comment:
FYI, this isn't that different from the hacks I used in [`atom/language-xml#54`](https://github.com/atom/language-xml/pull/54) to circumvent runaway highlighting with `</script>` tags and stuff. Heck, that PR's hacks are a hell of a lot worse, and the results aren't even gonna be appreciated on the level that consistent JSDoc highlighting will.